### PR TITLE
Mage_CurrencySymbol - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- add missing properties

### CODE REVIEW

File: app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php [hunk 1]
```diff
@@ -33,6 +33,16 @@
  */
 class Mage_Currencysymbol_Block_Adminhtml_System_Currencysymbol extends Mage_Adminhtml_Block_Widget_Form
 {
+    /**
+     * @var string
+     */
+    private $_blockGroup;
```
Comment (Sekiphp) [inline app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php]: I think that _blockGroup, _controller should be protected or deleted. As I see in project this attributes is not used. From this class is not extended any class, so I think, that these variables are pointless.

Comment (sreichel) [reply app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php]: Fixed.

File: app/code/core/Mage/CurrencySymbol/Model/System/Currencysymbol.php [hunk 2]
```diff
@@ -30,6 +30,8 @@
  * @category    Mage
  * @package     Mage_CurrencySymbol
  * @author      Magento Core Team <core@magentocommerce.com>
+ *
+ * @method $this resetValues()
```
Comment (tmotyl) [inline app/code/core/Mage/CurrencySymbol/Model/System/Currencysymbol.php:34]: what is it used for? what does it mean?

Comment (sreichel) [reply app/code/core/Mage/CurrencySymbol/Model/System/Currencysymbol.php:34]: Seems to be wrong commited .... can't find this in my patches. I'll check ....

Comment (sreichel) [reply app/code/core/Mage/CurrencySymbol/Model/System/Currencysymbol.php:34]: It is "used" in `Mage_CurrencySymbol_Adminhtml_System_CurrencysymbolController::resetAction()`, but this method does not exist ... not sure why, but i guess it was just to supress warnings.

Comment (tmotyl) [reply app/code/core/Mage/CurrencySymbol/Model/System/Currencysymbol.php:34]: ok, we have bigger issue here. adding this doc comment doesn't hurt

**Pull request merged**